ansible/roles/installation/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Create shared volume directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    -  /home/ansible/config/credentials/
+    -  /home/ansible/traefik/
+- name: Add traefik admin user with htpasswd file
+  htpasswd:
+    path: /home/ansible/traefik/passwords
+    name: "{{ traefik.user }}"
+    password: "{{ traefik.password }}"
+- name: Copy rails credentials file
+  copy:
+    src: ../config/credentials/ansible.yml.enc
+    dest: /home/ansible/config/credentials/production.yml.enc
```

Comment:
What’s happening here? Why can we simply use the Ansible vault file as a Rails credentials file?